.gitattributes

```diff
@@ -0,0 +1,3 @@
+client/src/session/rest/api/common.ts linguist-generated=true
+client/src/session/rest/api/compute.ts linguist-generated=true
```

Comment:
For this directory is the intent that all of the files here would be generated typescript source? I'm just wondering about explicitly enumerating each file in the git attributes versus just denoting it for the entire directory.

Reply:
> For this directory is the intent that all of the files here would be generated typescript source? 

I'll defer to @scnwwu for the answer to this. The reason I chose to explicitly list each file is I didn't want to end up in a situation where a user put something in there that _isn't_ auto-generated, but that we were less apt to look at (since it's collapsed by default).

Also, this keeps the files from expanding in a pull request. However, they _still_ show up. This makes these files behave more like `yarn.lock` (and other well known generated files). So, while they aren't expanded by default, a reviewer can still expand to see _what_ was changed.

Reply:
I would expect this directory to contain only OpenAPI generated files.

Reply:
BTW., `server/data/**` and `server/pubsdata/**` are also generated files.